frontend/packages/dev-console/src/components/topology/TopologySideBar.tsx

```diff
@@ -4,13 +4,21 @@ import { CloseButton } from '@console/internal/components/utils';
 import { ResourceOverviewPage } from '@console/internal/components/overview/resource-overview-page';
 import { TopologyDataObject, ResourceProps } from './topology-types';
 import './TopologySideBar.scss';
+import {
+  DEPLOYMENTCONFIGKIND,
+  DEPLOYMENTKIND,
+  DAEMONSETKIND,
+  STATEFULSETKIND,
+} from './topology-utils';
 
 export type TopologySideBarProps = {
   item: TopologyDataObject;
   show: boolean;
   onClose: Function;
 };
 
+const possibleKinds = [DEPLOYMENTCONFIGKIND, DEPLOYMENTKIND, DAEMONSETKIND, STATEFULSETKIND];
```

Comment:
I think this should be moved to topology-utils as well. Also, I am thinking if we can create an enum of kinds instead of using an array, WDYT?

Reply:
`possiblekinds` are only used in the sidebar. So it made sense for me to keep it in the sidebar but yeah we should create an enum for the kinds we use even then we would need an array.

Reply:
Hmm, but we might have a similar use-case for it elsewhere in future, so better to keep it in a commonplace. Better to keep it in `topology-types`. Also, create an enum for this as it will be much cleaner and less error-prone and also since these are just possible values for a single variable `kind` doesn't make sense to create 4 different strings and importing them.

Reply:
https://github.com/openshift/console/pull/2162/commits/6e1b2bfc91fa4abfbf7f7a124df61c4884702c89